unittests/UnitTestFramework.xml

```diff
@@ -211,50 +217,56 @@ declare
 enddeclare
 begin
 	p = ProcExec(CurMethod.Parent);
-	if expectedInteger is not null and actualInteger is not null then
-		if expectedInteger = actualInteger then
+	if expectedInteger is not null then
+		if actualInteger is not null or expectedInteger = actualInteger then

```

Comment:
I think this is wrong - this would throw an error for all non-nullable actualInteger values.
Likewise for the other cases (actualDate, etc.).

Reply:
Thanks, I have corrected the condition.